script/hatchet_config

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```

Comment:
I think the idea behind having just a `hatchet.ini` in the `script/` folder (as you'll see is in the `master` branch), was to just have these `ini` files that are good for certain scenarios, and can all just be run with `hatchet run <inifile` (we can have as many as we want and comment them copiously) and not have any bash scripts at all. If this can be done, that might be ideal.

Reply:
Looks like this bash script is gone now..